cupy/cuda/cupy_cuda.h

```diff
@@ -269,6 +269,26 @@ cudaError_t cudaDeviceEnablePeerAccess(...) {
     return cudaSuccess;
 }
 
```

Comment:
Need to also add stubs for `cudaIpcMemHandle_t` and `cudaIpcEventHandle_t` here.

Reply:
(may be also `cudaIpcMemLazyEnablePeerAccess`?)